build/common/installer/scripts/fluent-bit-conf-customizer.rb

```diff
@@ -115,6 +145,33 @@ def substituteFluentBitPlaceHolders
       new_contents = new_contents.gsub("\n    ${TAIL_IGNORE_OLDER}\n", "\n")
     end
 
+    if is_high_log_scale_mode?
```

Comment:
consider defining a function as the code is similar here: 
new_contents = update_config(new_contents, "TAIL_THREADED", "threaded on", enableFbitThreading)
new_contents = update_config(new_contents, "STORAGE_TYPE", "storage.type " + @default_high_log_scale_max_storage_type, storageType)
new_contents = update_config(new_contents, "MAX_STORAGE_CHUNKS_UP", "storage.max_chunks_up " + @default_high_log_scale_max_storage_chunks_up, storageMaxChunksUp)